test/benchmark/main.cc

```diff
@@ -0,0 +1,26 @@
+// NOLINT(namespace-envoy)
+// This is an Envoy driver for benchmarks.
+
+#ifdef ENVOY_HANDLE_SIGNALS
+#include "common/signal/signal_action.h"
+
```

Comment:
Nit: gratuitous blank line.

Reply:
fixed.

Reply:
Fix format is what is adding this newline, it's not me.